### PR TITLE
37 foundations react

### PR DESCRIPTION
**Pull request by BovkunLeonid**

### CODE REVIEW

File: 37_foundations_react/foundations-react/src/header/Header.jsx
Comment (RomanIlkiv) [inline 37_foundations_react/foundations-react/src/header/Header.jsx:1]: name of folder should be also from capital letter
it's a convention of React

File: 37_foundations_react/foundations-react/src/products/Products.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import './products.css';
+
+const products = [
+  {name: 'UMK start', id: 1, price: 100, alt: 'max-start', url: 'https://parettool.net/image/cache/catalog/product/mebelnyj-konduktor/UMK/%20%D0%BC%D1%96%D0%BD%D1%96/UMKM01-800x800.jpg'},
+  {name: 'UMK kit', id: 2, price: 150, alt: 'max-kit', url: 'https://parettool.net/image/cache/catalog/product/mebelnyj-konduktor/UMK/%20%D1%81%D0%B5%D1%80%D0%B5%D0%B4%D0%BD%D1%8F/UMKS22-800x800.jpg'},
+  {name: 'UMK max', id: 3, price: 200, alt: 'max-foto', url: 'https://parettool.net/image/cache/catalog/product/mebelnyj-konduktor/UMK/%20%D1%81%D0%B5%D1%80%D0%B5%D0%B4%D0%BD%D1%8F/UMKS12-800x800.jpg'},
+];
+
+const listItems = products.map((product) =>
+  <div className='item'>
```
Comment (RomanIlkiv) [inline 37_foundations_react/foundations-react/src/products/Products.jsx:10]: move this HTML to a separate component `Product`

this component will grow in the future, it should be separated

File: 37_foundations_react/foundations-react/src/products/Products.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <div>{product.name}</div>
+    <img src={product.url} alt={product.alt} />
+    <div>{product.price} $</div>
+  </div>
+);
+
+export default function Products() {
+  return (
+    <div className='products'> {listItems}  </div>
```
Comment (RomanIlkiv) [inline 37_foundations_react/foundations-react/src/products/Products.jsx:19]: no spaces inside of tag

Comment (RomanIlkiv) [reply 37_foundations_react/foundations-react/src/products/Products.jsx:19]: and moving it to a variable is not according to React convention

I know that it maybe more convenient for you to read, but we have to follow React community  conventions

File: 37_foundations_react/foundations-react/src/header/Header.jsx
Comment (BovkunLeonid) [reply 37_foundations_react/foundations-react/src/header/Header.jsx:1]: OK thank you

File: 37_foundations_react/foundations-react/src/products/Products.jsx (hunk 2 above)
Comment (BovkunLeonid) [reply 37_foundations_react/foundations-react/src/products/Products.jsx:19]: OK thank you

File: 37_foundations_react/foundations-react/src/products/Products.jsx (hunk 1 above)
Comment (BovkunLeonid) [reply 37_foundations_react/foundations-react/src/products/Products.jsx:10]: OK thank you

**Pull request merged**